examples/call_between_rust_and_python.rs

```diff
@@ -0,0 +1,88 @@
+use rustpython_vm::{
+    builtins::PyStr,
+    function::{FuncArgs, KwArgs, PosArgs},
+    pyclass, pymodule, PyObject, PyObjectRef, PyPayload, PyResult, TryFromBorrowedObject,
+    VirtualMachine,
+};
+
+pub(crate) use rust_py_module::make_module;
+
+pub fn main() {
+    let interp = rustpython_vm::Interpreter::with_init(Default::default(), |vm| {
+        vm.add_native_modules(rustpython_stdlib::get_module_inits());
+        vm.add_native_module("rust_py_module".to_owned(), Box::new(make_module));
+    });
+
+    interp.enter(|vm| {
+        vm.insert_sys_path(vm.new_pyobj("examples"))
+            .expect("add path");
+
+        let module = vm.import("call_between_rust_and_python", None, 0).unwrap();
+        let init_fn = module.get_attr("python_callback", vm).unwrap();
+
+        vm.invoke(&init_fn, ()).unwrap();
+
+        let pystr = PyObjectRef::from(PyStr::new_ref(
+            unsafe {
+                PyStr::new_ascii_unchecked(String::from("Rust string sent to python").into_bytes())
+            },
+            vm.as_ref(),
+        ));
+        let take_string_args = FuncArgs::new(PosArgs::new(vec![pystr]), KwArgs::default());
+        let take_string_fn = module.get_attr("take_string", vm).unwrap();
+
+        vm.invoke(&take_string_fn, take_string_args).unwrap();
+    })
+}
+
+#[pymodule]
+mod rust_py_module {
+    use super::*;
+
+    #[pyfunction]
+    fn rust_function(
+        num: i32,
+        s: String,
+        python_person: PyObjectRef,
```

Comment:
could just be `PythonPerson`, the conversion will happen automatically.

Reply:
Done.